public/javascripts/ng-controllers.js

```diff
@@ -154,7 +153,31 @@ springInitiative.controller('settingsController', function($scope, $rootScope, $
       username.isAdmin = !username.isAdmin;
     })
     .error(function(data) {
-      console.log('Error occured while admin change');
+      console.log('Error: ' + data)
     });
   };
+
+  $scope.changePassword = function(user) {
+    if($scope.currentPassword1 === $scope.currentPassword2 && $scope.currentPassword1 != undefined){
```

Comment:
The user will be asked to enter password twice in the form. So if they entered the right thing twice, the password changes. Else, enter again.